src/pages/workflows/workflow/workflows-acl-utils.ts

```diff
@@ -0,0 +1,16 @@
+export interface RawWorkflowsPermissions {
```

Comment:
I find `Raw` in the name a little confusing. What do you think about `WorkflowUserPermissions` or `WorkflowPermissionDetails`?

Reply:
I also found the name confusing at first, but I was not sure if there was a reason for it related to the permissions structure in Agora. I can definitely change it to `WorkflowUserPermissions`.